### ISSUE TITLE
[Bug]: Wither storm invisible on water 

### ISSUE THREAD

**Issue by kapycta123456** [nonamecrackers2/crackers-wither-storm-mod#1669]
### Minecraft Version

1.20.1

### Mod Version

4.0.1

### Mod Loader

Forge

### Mod Loader Version

don't rlly remember

### Describe the bug

Soo wither storm isn't seen on water with Complementary Shaders V4.7.1 (idk if this version of shaders is old)

### Steps to Reproduce

1. Play on 1.20.1 with Cracker's wither storm Evolution part 1 update
2. Get complementary shaders V4.7.1
3. Click it
4. Look down to the water when wither storm is near you
5. bug

### Crash Reports/Logs

NaN (no crash)

### Is OptiFine installed?

- [ ] OptiFine is installed

### Screenhots

![2023-12-08_21 34 17](https://github.com/nonamecrackers2/crackers-wither-storm-mod/assets/148063104/0ea78bf0-ff30-4d9d-8f83-d16981be9e8a)

### Mod List

Cracker's Wither storm
Rubidium
Oculus
Shader pack: Complementary shaders V4.7.1

**Comment by TaintedBlockmObile** [comment]
Might just be the shaders causing it

**Comment by kapycta123456** [comment]
> Might just be the shaders causing it

Yeah they should fix it i guess

**Comment by Aimyx** [comment]
is this is the same issue that causes tractor beams to not render through
glass and water? like the base game no shader issue?

**Comment by TaintedBlockmObile** [comment]
Yes and no. This is purely because of the shaders and how it and the wither storm renders

**Comment by nonamecrackers2** [comment]
I believe this happens without shaders as well. This is probably related to the stained glass issue, where the Wither Storm disappears when viewed behind stained glass.

**Comment by Herobrine982950291** [comment]
NaN is '' *N*ot A *N*umber ''
